vmdb/spec/requests/api/collections_spec.rb

```diff
@@ -0,0 +1,186 @@
+#
+# Rest API Collections Tests
+#
+require 'spec_helper'
+
+describe ApiController do
+
+  include Rack::Test::Methods
+
+  let(:api_user)   { @cfme[:user] }
+  let(:zone)       { FactoryGirl.create(:zone, :name => "api_zone") }
+  let(:miq_server) { FactoryGirl.create(:miq_server, :zone => zone) }
+  let(:template) do
+    FactoryGirl.create(:miq_template, :name => "template 1", :vendor => "vmware", :location => "template1.vmtx")
```

Comment:
do you need to set the default values for `:template`?
Shouldn't they just be set in the factory for us?
Are they even needed?